app/businessrates/authorisation/config/FeatureSwitch.scala

```diff
@@ -14,13 +14,18 @@
  * limitations under the License.
  */
 
-package businessrates.authorisation.connectors
+package businessrates.authorisation.config
 
-import businessrates.authorisation.models.Person
+import play.api.Configuration
 
-import scala.concurrent.{ExecutionContext, Future}
-import uk.gov.hmrc.http.HeaderCarrier
+import javax.inject.Inject
 
-trait PersonAccounts {
-  def getPerson(externalId: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Option[Person]]
+class FeatureSwitch @Inject()(config: Configuration) {
+
+  private val prefix = "feature-switch"
```

Comment:
while there's nothing wrong with "feature-switch" prefix, I would prefer just "feature"
it's configuring/enabling a specific feature. The "switch" is the mechanism that uses this config to do the switching.

Reply:
I know, you have already raised 10 other PRs with config changes 😬 so I guess you will not want to change it now. But I would still like that